wordpress/wp-content/plugins/sil-dictionary-webonary/include/configuration.php

```diff
@@ -296,6 +296,41 @@ function webonary_conf_widget($showTitle = false)
 	if(isset($_POST['btnSaveRelevance']))
 		relevanceSave();
 
+	$import_status = '';
+	$arrLanguageCodes = array();
+	$noReversalEntries = true;
+	if (get_option('useCloudBackend')) {
+		$dictionary_id = Webonary_Cloud::getBlogDictionaryId();
+		$dictionary = Webonary_Cloud::getDictionary($dictionary_id);
+		$import_status = '';
+		if (!is_null($dictionary)) {
+			$import_status .= '<li>Last Import: <em>' . $dictionary->updatedAt . '</em>';
+			$import_status .= '<li>Main Language (' . $dictionary->mainLanguage->lang . ')';
+			$import_status .= ' entries: <em>' . number_format($dictionary->mainLanguage->entriesCount) . '</em>';
+			$arrLanguageCodes[] = array(
+				'language_code' => $dictionary->mainLanguage->lang,
+				'name' => $dictionary->mainLanguage->title ?? $dictionary->mainLanguage->lang);
+			foreach ($dictionary->reversalLanguages as $reversal) {
+				$import_status .= '<li>Reversal Language (' . $reversal->lang . ')';
+				if (isset($reversal->entriesCount) && $reversal->entriesCount) {
+					$import_status .= ' entries: <em>'. number_format($reversal->entriesCount) . '</em>';
+					$arrLanguageCodes[] = array(
+						'language_code' => $reversal->lang,
+						'name' => $reversal->title ?? $reversal->lang);
+					$noReversalEntries = false;
+				}
+			}
+		}
+		$import_status = '<ul>' . $import_status . '</ul>';
+	}
+	else {
+		$import_status = Webonary_Info::import_status();
+		$arrLanguageCodes = Webonary_Configuration::get_LanguageCodes();
+		$displayXHTML = true;
+		$reversalEntries = getReversalEntries("", 0, "", $displayXHTML, "");
```

Comment:
It appears the function `getReversalEntries` is doing 2 different things. It should be refactored, but that's a different task.

Reply:
Yes, I noticed that, too. Awaiting refactoring later...